[@types/node] error TS2430: Interface 'NodeJS.Module' incorrectly extends interface '__WebpackModuleApi.Module'.

- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for Gitter.
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @jkomyno @a-tarasyuk @alvis @r3nya @btoueg @brunoscheufler @smac89 @touffy @DeividasBakanas @eyqs @Flarna

```bash
node_modules/@types/node/ts3.1/globals.d.ts:573:15 - error TS2430: Interface 'NodeJS.Module' incorrectly extends interface '__WebpackModuleApi.Module'.
  Types of property 'parent' are incompatible.
    Type 'Module | null | undefined' is not assignable to type 'NodeModule | null'.
      Type 'undefined' is not assignable to type 'NodeModule | null'.

573     interface Module {
                  ~~~~~~
```

--- a/types/webpack-env/index.d.ts
+++ b/types/webpack-env/index.d.ts
@@ -68,7 +68,7 @@ declare namespace __WebpackModuleApi {
         id: string;
         filename: string;
         loaded: boolean;
-        parent: NodeModule | null;
+        parent: NodeModule | null | undefined;
         children: NodeModule[];
         hot?: Hot;
     }

--- a/types/webpack-env/webpack-env-tests.ts
+++ b/types/webpack-env/webpack-env-tests.ts
@@ -1,4 +1,4 @@
-
+import 'node';
 
 interface SomeModule {
     someMethod(): void;

--- a/types/webpack-env/webpack-env-tests.ts
+++ b/types/webpack-env/webpack-env-tests.ts
@@ -1,4 +1,4 @@
-import 'node';
+/// <reference types="node"/>
 
 interface SomeModule {
     someMethod(): void;

